packages/docs/source/components/status.md

```diff
@@ -0,0 +1,211 @@
+    </dt>
+    <dd class="ods-status--value">
+      System inactive
+    </dd>
+  </dl>
+  ```
+</figure>
+
+### Success
+
+Success Statuses are green and should be used to indicate states like Complete, Active, Available, Service Operational.
+
+<figure class="nimatron--example">
+  <div class="nimatron--rendered">
+    <dl class="ods-status is-ods-status-success">
+      <dt class="ods-status--label">
+        Server status
+      </dt>
+      <dd class="ods-status--value">
+        System operational
+      </dd>
+    </dl>
+  </div>
+
+  ```html
+  <dl class="ods-status is-ods-status-success">
+    <dt class="ods-status--label">
+      Server status
+    </dt>
+    <dd class="ods-status--value">
+      System operational
+    </dd>
+  </dl>
+  ```
+</figure>
+
+### Caution
+
+Caution Statuses are yellow and should be used to indicate states like Attention Suggested or Service Degradation.
+
+<figure class="nimatron--example">
+  <div class="nimatron--rendered">
+    <dl class="ods-status is-ods-status-caution">
+      <dt class="ods-status--label">
+        Server status
+      </dt>
+      <dd class="ods-status--value">
+        Service degradation
+      </dd>
+    </dl>
+  </div>
+
+  ```html
+  <dl class="ods-status is-ods-status-caution">
+    <dt class="ods-status--label">
+      Server status
+    </dt>
+    <dd class="ods-status--value">
+      Service degradation
+    </dd>
+  </dl>
+  ```
+</figure>
+
+### Danger
+
+Danger Statuses are red and should be used to indicate states like Error, Failure, or Service Disruption.
+
+<figure class="nimatron--example">
+  <div class="nimatron--rendered">
+    <dl class="ods-status is-ods-status-danger">
+      <dt class="ods-status--label">
+        Server status
+      </dt>
+      <dd class="ods-status--value">
+        Service Disruption
+      </dd>
+    </dl>
+  </div>
+
+  ```html
+  <dl class="ods-status is-ods-status-danger">
+    <dt class="ods-status--label">
+      Server status
+    </dt>
+    <dd class="ods-status--value">
+      Service Disruption
+    </dd>
+  </dl>
+  ```
+</figure>
+
+## Content guidelines
+
+Status is intended to be both succinct and easily understood; limit label and description to one to two words. As with other components, use sentence casing for both.
+
+## Accessibility
+
+In addition to including copy for assistive technologies, consider whether your Status may change while the user is on the page.
+
+If the current state of Status may change asynchronously while a user is visiting the page, utilize the `role="status"` attribute to ensure that assistive technologies correctly indicate this change.
+
+<strong>Note:</strong> This attribute must be present <em>before</em> the change occurs.
+
+<figure class="nimatron--example">
+  <div class="nimatron--rendered">
+    <dl class="ods-status is-ods-status-danger">
+      <dt class="ods-status--label">
+        Server status
+      </dt>
+      <dd class="ods-status--value" role="status">
```

Comment:
@arnoldsandoval-okta thoughts on where to place `role`? The `dl` represents the component as a whole, but the value is where the async update will occur. 

This feels semantically rough, but probably safer for a11y?

Reply:
Placing it at the top level should be fine

Reply:
Just realized we want the label to read when it updates anyway. 👍

Reply:
Yeah, I think of the entire component as the status region, not any one part of it.